sql/src/main/java/io/crate/metadata/view/ViewsMetaData.java

```diff
@@ -163,6 +163,10 @@ public int hashCode() {
         return Objects.hash(viewByName);
     }
 
+    public boolean contains(RelationName relationName) {
+        return contains(relationName.fqn());
+    }
+
     public boolean contains(String name) {
```

Comment:
Can we remove remove this `contains` overload and only have the one that requires a `RelationName`?

During the initial implementation we had a mix up of `fqn()` and `sqlFqn()`. By always requiring a `relationName` we have all calls isolated inside this class